Sources/Collar/Classes/List/SwiftUI/LogListView.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  LogListView.swift
+//  Pods
+//
+//  Created by Petar Jadek on 30.09.2025.
+//  Copyright © 2025 Infinum. All rights reserved.
+//
+
+import SwiftUI
+
+struct LogListView: View {
+
+    @Environment(\.dismiss) private var dismiss
+
+    private let notificationName = AnalyticsCollectionManager.Notification.didUpdateLogs.name
+    private let analyticsManager = AnalyticsCollectionManager.shared
+
+    @State private var items: [LogItem] = []
+    @State private var searchText = ""
+    @State private var showFilters = false
+    @State private var selectedFilterType: LogFilterType = .all
+
+    var filteredItems: [LogItem] {
+        items.filter { item in
+            let matchesSearch: Bool
+            if searchText.isEmpty {
+                matchesSearch = true
+            } else {
+                matchesSearch =
+                item.name.localizedCaseInsensitiveContains(searchText) ||
+                (item.subtitleDisplay ?? "").localizedCaseInsensitiveContains(searchText)
+            }
```

Comment:
New swift syntax should allow something like this
```suggestion
            let matchesSearch: Bool = if searchText.isEmpty {
                true
            } else {
                item.name.localizedCaseInsensitiveContains(searchText) ||
                (item.subtitleDisplay ?? "").localizedCaseInsensitiveContains(searchText)
            }
```